lib/timelines.rb

```diff
@@ -1,6 +1,7 @@
 require "timelines/version"
 require "timelines/railtie"
-require "timelines/ephemeral"
+require "timelines/models/concerns/ephemeral"
+require "timelines/models/event"
```

Comment:
You also need to require your new concern `require "timelines/models/concerns/has_events"`